fetch/datablobstorage/copy_direct.go

```diff
@@ -52,7 +52,9 @@ func (c *copyCRDBDirect) CreateFromReader(
 	}
 
 	c.logger.Debug().Int("batch", iteration).Msgf("csv batch starting")
-	if _, err := conn.PgConn().CopyFrom(ctx, r, dataquery.CopyFrom(table, false /*skipHeader*/)); err != nil {
+	query := dataquery.CopyFrom(table, false /*skipHeader*/)
+	c.logger.Info().Msgf("runnign copy drom: %s", query)
```

Comment:
```suggestion
	c.logger.Info().Msgf("runnign copy from: %s", query)
```
This shouldn't log anything sensitive out right? We redacted the one for import since i had credentials. But copy from should be safe right?

Reply:
We should also make this debug level to match the import level.

Reply:
Yes, `COPY FROM` stmt only contains the table and column names so should be fine.